### ISSUE TITLE
Why can't it parse JSX syntax?

### ISSUE THREAD

**Issue by mayu888** [babel/babel#12444]
我自己写了一个用于处理js的插件，在react项目中使用，插件中也使用了inherits来处理jsx语法，但是它为什么还是报错呢？
I have written a plug-in to handle JS, which is used in react project. Inheritance is also used in the plug-in to process JSX syntax. But why does it still report errors?
![image](https://user-images.githubusercontent.com/52163863/101185196-86cf8380-368c-11eb-96f2-e9a00da24010.png)
![image](https://user-images.githubusercontent.com/52163863/101185258-9a7aea00-368c-11eb-9489-b4695832cb88.png)
![image](https://user-images.githubusercontent.com/52163863/101185281-a23a8e80-368c-11eb-8a12-641f0873c014.png)

Is there a problem with my use?

**Comment by nicolo-ribaudo** [comment]
The problem is that you are only telling Babel how to parse JSX. Babel parser JSX, doesn't transform it (because you haven't enabled a plugin to transform JSX), and then gives JSX back to webpack.
Webpack doesn't know how to handle JSX, so it throws.

**Comment by mayu888** [comment]
So should I use the @ Babel / plugin transform react JSX plug-in to solve this problem？So how to do it?This is my first time to write a plug-in. I hope you can enlighten me more~

**Comment by mayu888** [comment]
[here](https://github.com/mayu888/musicByStages- )，You can watch **”babel-add-try-catch- loader.js“**  This document。
This is my first time to write plug-ins, I hope you can give me more advice~thanks

**Comment by mayu888** [comment]
thanks，I've solved this bug。But I have another problem now，I found that my own plug-ins and official plug-ins are executed in parallel，In my own plug-in, when the function I write runs, the code has been modified by other plug-ins. My plug-in can't handle the code correctly. The document does not find out how to determine the execution order of each plug-in. What should I do to enable my plug-ins to execute after all the plug-ins are executed?
